lib/avrolution/configuration.rb

```diff
@@ -35,6 +40,13 @@ def self.compatibility_schema_registry_url
     end
   end
 
+  def self.deployment_schema_registry_url
+    @deployment_schema_registry_url ||= begin
+      raise 'deployment_schema_registry_url must be set' unless ENV[DEPLOYMENT_SCHEMA_REGISTRY_URL]
```

Comment:
Could do `ENV.fetch(DEPLOYMENT_SCHEMA_REGISTRY_URL) { raise 'deployment_schema_registry_url must be set' }`